### ISSUE TITLE
Not an error, but a request

### ISSUE THREAD

**Issue by Nekonaro** [Jboymercs/end_expansion_final#10]
Hi, thanks for the nice mod! If you have time, could you please add compatibility with Tinkers Construct and Construct's Armory? I don't know why you decided to make a mod for the version 1.12.2 forgotten by many players, but I want to say thank you. I love Minecraft 1.12.2

**Comment by Jboymercs** [comment]
Yeah I'll look into it, as for the answer to that second part, it's just my favorite version currently plus I'm too unbothered to move up with all the code changes. There is a community of 1.12.2 players if you haven't joined yet called the 1.12.2 Modding Coalition -> [https://discord.gg/2JZ8KePDKd](url). I guess to also fill you in on other stuff, I have other projects that are focused on the adventure/exploration side you can just check out my page on CF. Additionally, we have a Trial Chambers backport coming out soon along with the nether backport, so stay tuned for those as well!
